src/main/scala-2.12/com/linagora/gatling/imap/protocol/ImapProtocol.scala

```diff
@@ -13,12 +13,15 @@ import io.gatling.core.session.Session
 object ImapProtocol {
   val ImapProtocolKey = new ProtocolKey[ImapProtocol, ImapComponents] {
 
+    var protocolNumber: Int = 0
+
     override def protocolClass: Class[io.gatling.core.protocol.Protocol] = classOf[ImapProtocol].asInstanceOf[Class[io.gatling.core.protocol.Protocol]]
 
     override def defaultProtocolValue(configuration: GatlingConfiguration): ImapProtocol = throw new IllegalStateException("Can't provide a default value for ImapProtocol")
 
     override def newComponents(coreComponents: CoreComponents): ImapProtocol => ImapComponents = { protocol =>
-      val sessions: ActorRef = coreComponents.actorSystem.actorOf(ImapSessions.props(protocol), "imapsessions_" + randomUUID().toString)
+      val sessions: ActorRef = coreComponents.actorSystem.actorOf(ImapSessions.props(protocol), s"imapsessions-${protocolNumber}")
```

Comment:
But  a random UUID was already appended to the actor name ?

Reply:
Oh damn you are right... what happened and didn't understand yesterday was that I first saw this on james-gatling... that was using a completely outdated gatling-imap. I guess what I did then was just applying the change on gatling-imap without looking carefully enough to see that it has been fixed already (and in a better way)

Thanks for the feedback, I think I can close this PR